packages/babel/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@rollup/plugin-babel",
+  "version": "5.0.0-alpha.1",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Seamless integration between Rollup and Babel.",
+  "license": "MIT",
+  "repository": "rollup/plugins",
+  "author": "Rich Harris",
+  "homepage": "https://github.com/rollup/plugins/tree/master/packages/babel",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "./dist/rollup-plugin-babel.cjs.js",
+  "engines": {
+    "node": ">= 8.0.0"
+  },
+  "scripts": {
```

Comment:
`scripts` should be about the same as the other plugins

Reply:
what does it mean? what's missing here?

Reply:
this is an easy one - take a look at literally any of the other plugins. copy and paste, adjust for build where needed.